kijiji_repost_headless/__main__.py

```diff
@@ -102,6 +109,23 @@ def show_ads(args, api=None):
         api = kijiji_api.KijijiApi()
         api.login(args.username, args.password)
     all_ads = api.get_all_ads()
+
+    # Sort list of ads if a sort key was given
+    if args.sort_key:
+        key = args.sort_key
+        if key.lower() == 'id':
+            sort_key = 'id'
+        elif key.lower() == 'title' or key.lower() == 'name':
+            sort_key = 'title'
+        elif key.lower().startswith('rank') or key.lower().startswith('page'):
```

Comment:
What's the reason to allow 'page'? I think just 'rank' should be enough. Same commend for 'name' vs 'title'

If we only allow the keys that Kijiji has, you could probably just check (key.lower() in ['id', 'title', 'views']) and do the check much more concisely

Reply:
My thinking was that some users might not know what the 'rank' key is for, but 'page' is pretty clear that it means which page number the ad is on. Same idea for including 'name' as a synonym for the 'title' key.
I suppose as long as the key names are properly shown in the help menu, then there likely isn't a need to have much flexibility in the command line arguments. Perhaps in the future Kijiji might add another separate key called 'name' which would then be in conflict... I think what I'll do is make all the key name arguments more static and just match them to what Kijiji calls them internally.